data/blog/electron-5-0.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Electron 5.0.0
+author:
+- BinaryMuse
+- jkleinsc
+date: '2018-04-23'
+---
+
+The Electron team is excited to announce that the stable release of Electron 5 is now available! You can install it from [electronjs.org](https://electronjs.org/) or from npm via `npm install electron@latest`. The release is packed with upgrades, fixes, and new features, and we can't wait to see what you build with them. Read more for details about this release, and please share any feedback you have as you explore!
+
+---
+
+## What's New?
+
+A large part of Electron's functionality is provided by Chromium, Node.js, and V8, the core components that make up Electron. As such, a key goal for the Electron team is to keep up with changes to these projects as much as possible, providing developers who build Electron apps access to new web and JavaScript features. To this end, Electron 5 features major version bumps to each of these components; Electron v5.0.0 includes Chromium `73.0.3683.117`, Node `12.0.0`, and V8 `7.3.492.27`.
+
+In addition, Electron 5 includes changes to Electron-specific APIs. You can find a summary of the major changes in Electron 5 below; for the full list of changes, check out the [Electron v5.0.0 release notes](https://github.com/electron/electron/releases/tag/v5.0.0).
+
+### Promisification
+
+Electron 5 contains changes as part of our [Promisification](https://github.com/electron/electron/blob/master/docs/api/modernization/promisification.md) initiative to convert callback-based functions in Electron to return Promises.  The following APIs were converted for Electron
```

Comment:
This link should point to a commit hash or the Electron 5 branch

Reply:
Fixed dab62bf22db34855aacd3f9b6805dd093f11eedc